save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -158,6 +143,10 @@ class VulnerabilityService(
                 cb.equal(root.get<VulnerabilityLanguage>("language"), language)
             } ?: cb.and()
 
+            val chosenStatusPredicate = chosenStatus?.let { status ->
+                cb.equal(root.get<VulnerabilityStatus>("status"), status)
```

Comment:
I thought about this, but in the end I decided to highlight the difference between these two statuses by adding the description `Uploaded` for `AUTO_APPROVED`. I thought that it might not be obvious to user why some of the vulnerabilities have `Created` status after creation and require approval, while the other part loaded using files is immediately in the `APPROVED` status (now in `AUTO_APPROVED`).

Reply:
Done. These statuses are now displayed together under a common description `Approved`.